src/lib/api.ts

```diff
@@ -59,8 +57,8 @@ export default async function API( { exitOnError = true } = {} ): Promise<
 		}
 	} );
 
-	const withToken = setContext( async (): Promise< { token: Token } > => {
-		const token = await Token.get();
+	const withToken = setContext( async (): Promise< { token: Token | { raw: string } } > => {
+		const token = customAuthToken ? { raw: customAuthToken } : await Token.get();
```

Comment:
this types are needlessly complicated / gross. this suggestion has a companion a few lines down

```suggestion
	const withToken = setContext( async (): Promise< { token: string } > => {
		const token = customAuthToken || ( await Token.get() ).raw;
```

Reply:
actually gonna jsut push it up bc github won't let me suggest on unchanged code